src/hash/rescue/rpx/digest.rs

```diff
@@ -118,26 +118,106 @@ impl Randomizable for RpxDigest {
 // CONVERSIONS: FROM RPX DIGEST
 // ================================================================================================
 
-impl From<&RpxDigest> for [Felt; DIGEST_SIZE] {
-    fn from(value: &RpxDigest) -> Self {
-        value.0
+#[derive(Copy, Clone, Debug)]
+pub enum RpxDigestError {
+    InvalidInteger,
+}
```

Comment:
One thing I noticed: I don't think this or the `RpoDigestError` are exported from the crate. This probably doesn't cause any immediate issues, but may be a good idea to export them.